README.rst

```diff
@@ -86,23 +86,31 @@ Basic parameters
 
 The parameters below are not strictly required. It is recommended to set these parameters as they prove to be extremely useful to categorize your metrics and search by them.
 
-+--------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------+----------+
-| Name                           | Description                                                                                                                                                                                                                                                                                                                                                                                                                                               | Type                                                         | Required |
-+--------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------+----------+
-| SourceName                     | Name of the metrics source. ``_sourceName`` can be used to search metrics from this source. It will override the default configured in the the Sumo Logic Source configuration.                                                                                                                                                                                                                                                                           | String                                                       | False    |
```

Comment:
Why are these being removed?

Reply:
Oops, replaced the wrong table 😓 , thanks for catching it!